src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,8 @@
+$hover-color: #00ACDC;
+$color-comet: #616070;
```

Comment:
Use the same name styling
`$hover-color` and `$comet-color`
or `$color-hover` and `$color-comet`